src/main/java/org/fundacionjala/coding/oscar/movies/Movie.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.coding.oscar.movies;
+
+/**
+ * The class for represent a movie.
+ */
+public abstract class Movie {
+    private String title;
+    private int priceCode;
```

Comment:
If you remove the constants for CHILDREN, NEW_RELEASE, REGULAR, this variable will not be necessary anymore.